src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -213,6 +216,15 @@ public void UpdateDescriptionForModel(MappingDataFormatParameter mappingSource)
          else //in this case the column is a measurement column
          {
             column.LloqColumn = _mappingParameterEditorPresenter.LloqFromColumn() ? _mappingParameterEditorPresenter.LloqColumn : null;
+            if (dimensionChanged)
```

Comment:
Move into a separate method. Also make sure if that have more than one line have { }